### PR TITLE
chore: added tooltip for vacuum button

### PR DESCRIPTION
**Pull request by MehadND**
A lot of users were confused about the use of 'Vacuum' button in the settings->system. Therefore I added a tooltip for this.

At the moment the tooltip text is only in English language.

From [issue #2724](https://github.com/usememos/memos/issues/2724)

### CODE REVIEW

File: web/src/locales/en.json [hunk 1]
```diff
@@ -256,6 +256,7 @@
       "disable-public-memos": "Disable public memos",
       "max-upload-size": "Maximum upload size (MiB)",
       "max-upload-size-hint": "Recommended value is 32 MiB.",
+      "vacuum-hint": "Cleans up unused or obsolete data.",
```
Comment (jxpsert) [inline web/src/locales/en.json:259]: I think obsolete is (nearly) a synonym of unused. I personally think only 'unused' would look better.

**Review by jxpsert — COMMENTED** [review]
> At the moment the tooltip text is only in English language.

Was afraid you'd hardcoded the text from reading this, but you used i18n- good!

One comment on the text itself- not entirely important.
Code looks good, though!

File: web/src/locales/en.json (hunk 1 above)
Comment (MehadND) [reply web/src/locales/en.json:259]: Ok, thanks. Appreciate it

**Review by boojack — APPROVED** [review]
LGTM, thanks for your contribution!

**Pull request merged**